pkg/watcher/skr_webhook_manifest_manager.go

```diff
@@ -151,20 +131,51 @@ func (m *SKRWebhookManifestManager) Install(ctx context.Context, kyma *v1beta2.K
 	return nil
 }
 
+func (m *SKRWebhookManifestManager) verifyCertNotRenew(certificate *certmanagerv1.Certificate,
+	kyma *v1beta2.Kyma,
+) {
+	if certificate.Status.RenewalTime != nil &&
+		time.Now().Add(-m.certificateConfig.RenewBuffer).After(certificate.Status.RenewalTime.Time) {
+		m.WatcherMetrics.SetCertNotRenew(kyma.Name)
+	} else {
+		m.WatcherMetrics.CleanupMetrics(kyma.Name)
+	}
+}
+
+func verifyCACertRotation(ctx context.Context,
```

Comment:
This could be a method on the `CertificateManager` instead

Reply:
It also does not really verify, it actually removes the secret when rotated. Can you rename it?